gui/src/components/logs/FormattedLogMessage.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  export let message: string = '';
+
+  let words = message.split(' ');
+
+  const linkRegex = /^https*:\/\/\S+\.\S+/;
+</script>
+
+<span>
+  {#each words as word}
+    {#if linkRegex.test(word)}
+      <a href={word}>{word}</a>
+    {:else}
+      <span>{word}</span>
+    {/if}
+    {' '}
```

Comment:
This will _always_ include a trailing space. This might not be a problem, but could affect word wrapping or similar. If you do the parse loop solution mentioned above, the correct amount of spaces will be encoded in the spans, so you won't need to manually re-insert it.